docs/api-docs/api-use-cases.md

```diff
@@ -0,0 +1,300 @@
+        </li>
+    </ul>
+  </tr>
+</tbody>
+</table>
+
+<table id="sequencing-order" class="tg">
+<thead>
+  <tr>
+    <th class="tg-0lax">Use case #</th>
+    <th class="tg-0lax">4</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td class="tg-0lax">Name</td>
+    <td class="tg-0lax">Dashboard - Specific sequencing order status</td>
+  </tr>
+  <tr>
+    <td class="tg-0lax">Actors</td>
+    <td class="tg-0lax">
+        <ul>
+            <li>
+                BCDC service account
+            </li>
+            <li>
+                Sequencing tracker
+            </li>
+        </ul>
+    </td>
+  </tr>
+  <tr>
+    <td class="tg-0lax">Description</td>
+    <td class="tg-0lax">
+         <ul>
+            <li>
+                PI sees a particular order for their lab that doesn't match expected status. They want to see more information about the order to see why the status doesn't match.
+            </li>
+            <li>
+                The BCDC service account queries specimen portal API, maintaining an event log of order status. Events include BICAN identifier and/or URI for orders that allow redirection to sequencing portal for specific information about order. Dashboard reduces event log, displays specific order and links out to the sequencing portal to view details.
+            </li>
+        </ul>
+  </tr>
+</tbody>
+</table>
+
+<table id="cohesive-view" class="tg">
+<thead>
+  <tr>
+    <th class="tg-0lax">Use case #</th>
+    <th class="tg-0lax">5</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td class="tg-0lax">Name</td>
+    <td class="tg-0lax">Dashboard - One page view of specimen + library + sequencing artifacts</td>
+  </tr>
+  <tr>
+    <td class="tg-0lax">Actors</td>
+    <td class="tg-0lax">
+        <ul>
+            <li>
+              BCDC service account
+            </li>
+            <li>
+              Sequencing tracker
+            </li>
+            <li>
+              Specimen portal
+            </li>
+        </ul>
+    </td>
+  </tr>
+  <tr>
+    <td class="tg-0lax">Description</td>
+    <td class="tg-0lax">
+         <ul>
+            <li>
+                PI needs to find donor information for a specific sequenced cell from a specific order.
+            </li>
+            <li>
+                The BCDC service account queries specimen portal and sequencing portal API, maintaining an event log of order metadata. Order metadata includes identifiers and URIs for specimen. Specimen resources are queried to retrieve donor ID, and user is redirected to donor page.
+            </li>
+        </ul>
+  </tr>
+</tbody>
+</table>
+
+
+## Write use cases
+The following use cases describe scenarios where a downstream system/person needs to write information back to a given portal's system via API.
```

Comment:
Same thing with the "Write" use cases for GraphQL vs REST. The API would require less maintenance and users can update what they want with minimal data transfer.

Reply:
See comment on graphql